.Rbuildignore

```diff
@@ -2,6 +2,9 @@
 ^\.Rproj\.user$
 ^\.Renviron$
 ^\.github$
+^\.envrc$
+^\.vscode$
+^.*\.lic$
```

Comment:
I don't mind adding these, but IMO they should go separately (like as part of the version bump) rather than a feature PR. I'm fine to keep them here if that's quickest, but it's a funk we should avoid if possible

Reply:
Good point -- I'll move them to another PR.